src/cart/SConscript

```diff
@@ -60,7 +60,8 @@ def scons():
     SConscript('swim/SConscript')
     Default('swim')
 
-    Import('env', 'prereqs', 'swim_targets', 'gurt_lib', 'CART_VERSION')
+    Import('env', 'prereqs', 'swim_targets', 'gurt_lib', 'CART_VERSION',
+           'conf_dir')
```

Comment:
It's a bug in the scons pylint wrapper but if you split this into two Import statements, your warnings below will go away

Import('env', 'prereqs', ...)
Import('conf_dir')

Note that it doesn't complain about conf_dir in other places.

Reply:
Done.